README.md

```diff
@@ -1,28 +1,51 @@
-# spinnaker-kustomize-patches
+# Kustomize Patches for Armory
 
-This repository contains example [kustomize](https://kustomize.io) patch files to configure and deploy Spinnaker using the Spinnaker Operator.
+This repository contains example [Kustomize](https://kustomize.io) patch files to configure and deploy Armory using the [Armory Operator](https://docs.armory.io/docs/installation/operator/), which is a Kubernetes Operator for installing Armory.
 
-### Disclaimer
+The Armory Operator has a `basic` mode and a `cluster` mode. In `basic` mode, you configure Spinnaker in a single `spinnakerservice.yml` file. In `cluster` mode, though, you can configure with a single file **or** use Kustomize patch files. The advantage of using patch files is readability, consistency across environments, and maintenance manageability.  Instead of all the configuration sections in a very long `spinnakerservice.yml` file, each section is its own file, with a `kustomization.yml` file that uses the patch files to build a deployment file. See the [Managing Configuration](https://docs.armory.io/docs/installation/operator/#managing-configuration) section of the Spinnaker Operator docs for details and examples.
```

Comment:
The difference between `basic` and `cluster` operator modes is that `basic` doesn't do pre-flight checks/validations and `cluster` does. This repository installs operator in `cluster` mode when using `deploy.sh` script.

Both modes work with a single `spinnakerservice.yml` manifest or with kustomize patches.